src/expressions.md

```diff
@@ -88,9 +88,26 @@ created and used instead. The lifetime of temporary values is typically the
 innermost enclosing statement; the tail expression of a block is considered
 part of the statement that encloses the block.
 
-When a temporary rvalue is being created that is assigned into a `let`
-declaration, however, the temporary is created with the lifetime of the
-enclosing block instead, as using the enclosing statement (the `let`
+A first exception is when a temporary value is created in the
```

Comment:
The phrasing here seems odd and opens the possibility that there are more than two exceptions. Perhaps a paragraph stating "There are two exceptions to this rule." prior to the exceptions listings?